dtos/common/config.go

```diff
@@ -12,12 +12,14 @@ package common
 type ConfigResponse struct {
 	Versionable `json:",inline"`
 	Config      interface{} `json:"config"`
+	ServiceName string      `json:"serviceName"`
 }
 
 // NewConfigResponse creates new ConfigResponse with all fields set appropriately
-func NewConfigResponse(serviceConfig interface{}) ConfigResponse {
+func NewConfigResponse(serviceConfig interface{}, serviceName string) ConfigResponse {
```

Comment:
@lenny-intel @AlexCuse 
this is a breaking change
we should revert this PR
a new factory method should be added for this, such as NewXXXResponseWithServiceName
Or, the service name should be inserted to the DTO after it is created

Reply:
Thanks @cloudxxx8 new factory method is easy enough - where is the breaking change?

Reply:
@lenny-intel said it has been considered, so I am fine with it.
Basically, go-mod-core-contracts is a public module and NewConfigResponse is a public function.
If there is someone use NewConfigResponse in go-mod-core-contracts in their code, the compile error will happen once they upgrade the go-mod-core-contracts version

Reply:
Ah got it thanks @cloudxxx8 - my initial PR did have both but we removed on scope expansion to keep it simple.  Thanks!

Reply:
@cloudxxx8, @AlexCuse, I  Brought this up at today's TSC and we are ok with this breaking change since this code is meant to only be used by our services and SDKs. Risk of breaking someone else code is very low.